### ISSUE TITLE
Notes on fdp-ization

### ISSUE THREAD

**Issue by danfowler** [os-data/eu-allocations-per-ms-2014-2020#1]
@pwalsh as you say, there's not much to this one beyond being a set of measures. 

## Observations

- I added an array of countryCodes to reference all the European countries referenced in this dataset. Tedious, but I can see it would be useful.
- It seemed redundant to assign the same currency to each individual measure.
- It seemed redundant to assign a primaryKey on the schema as well as the logical model.  I see that you noted something similar here (https://github.com/openspending/fiscal-data-package/pull/52#issuecomment-144283935).  (I also see that the way I've written used primaryKey needs to be changed.)
- I set the fiscalPeriod to 2014 to 2020, which gave me pause, but I see that 2014-2020 is indeed a "programming period" for the EU.  It makes me wonder if a single dataset could represent multiple fiscalPeriods.

Does any of this strike you as something that I should raise as an issue on the main spec?

**Comment by pwalsh** [comment]
Yes, everything should be an issue for the main spec with the goals of the following outcomes (as required):

* iteration on spec
* discussion of spec scope in reference to data 'out there'
* variety in examples we will start to compile of the spec
